Lib/fontTools/varLib/merger.py

```diff
@@ -726,6 +728,31 @@ def _Lookup_PairPos_subtables_canonicalize(lst, font):
 
 	return lst
 
+def _Lookup_SinglePos_subtables_flatten(lst, font, min_inclusive_rec_format):
+	lst = list(lst)
```

Comment:
what's the purpose of the shallow copy `list(lst)` here? is it because you don't want to modify the original master fonts' SinglePos subtables?